### ISSUE TITLE
Can't get a response from my app when using the alexa developer console to test my skill

### ISSUE THREAD

**Issue by rv-ekramer** [jovotech/jovo-framework-nodejs#111]
I keep getting `There was a problem with the requested skills response.` when using jovo-cli version `1.0.10` and jovo-framework `1.0.4` or `1.1.1`. It doesn't matter whether I run `jovo run` or `jovo run --watch`. Nothing works.

**Comment by rv-ekramer** [comment]
Please let me know information you need to help debug this information.

**Comment by jankoenig** [comment]
Do you see any error in your command line?

**Comment by rv-ekramer** [comment]
Nope, the command line makes it seem like everything is fine 😢

**Comment by jankoenig** [comment]
Are you logging the requests and responses? So everything is apparently going in and out as it should? 

There could be a problem with the webhook then. For debugging, could you try `jovo run --bst-proxy` and see if it works with this endpoint?

**Comment by rv-ekramer** [comment]
I am logging the request and responses but since the app responds with "The requested skill took too long to respond" when telling it "open tv helper" I don't get any logs.

**Comment by rv-ekramer** [comment]
No luck

**Comment by jankoenig** [comment]
So there is no incoming request at all? Then it's most likely an endpoint problem. Few things to try:
* Make sure the endpoint in the Alexa Developer Console is right
* Replace it with e.g. the Bespoken proxy to see if it's a problem related to the Jovo Webhook

**Comment by rv-ekramer** [comment]
Correct there is no incoming request at all and the webhook URL in the alexa dev console is correct

![screen shot 2018-03-30 at 3 24 16 pm](https://user-images.githubusercontent.com/31253814/38150760-960ca41c-342e-11e8-8343-1cd129184422.png)
![screen shot 2018-03-30 at 3 24 21 pm](https://user-images.githubusercontent.com/31253814/38150763-97f05648-342e-11e8-9022-017a5252afb2.png)
![screen shot 2018-03-30 at 3 24 36 pm](https://user-images.githubusercontent.com/31253814/38150771-9af830d6-342e-11e8-9ce2-b468d55b9810.png)

Attached are screenshots of my terminal that states the url, the alexa dev console where the webhook url is placed and the alexa test simulator that has the response i've been getting

**Comment by rv-ekramer** [comment]
I ran `jovo run --bst-proxy` and `jovo run --watch` and `jovo run` no such luck with anything.

**Comment by jankoenig** [comment]
Hmm, I tried to reproduce it by creating a fresh new project with the newest CLI and Framework version. For me, it works. Two things I'd try:
* Delete the `node_modules` folder and do a new `npm install`
* Create a fresh new project with `jovo new` and see if it works there

**Comment by rv-ekramer** [comment]
I already tried removing the `node_modules` folder and re installing that didn't work either. But creating a new skill did. What does that mean and why does that work??

**Comment by rv-ekramer** [comment]
Also referencing #112 why can I add `jovo run --watch` to `npm run dev`?

**Comment by jankoenig** [comment]
Hmm, maybe it was a problem in the package.json that caused the issue? Can you show the relevant parts of it here?

**Comment by rv-ekramer** [comment]
The new skill was using version `1.0.0` of jovo and not `1.0.4` or `1.1.1` with the custom logging

**Comment by rv-ekramer** [comment]
package.json for the new project

```json
{
  "name": "jovo-sample-voice-app-nodejs",
  "version": "1.0.0",
  "description": "A sample voice app that works with the Jovo framework",
  "main": "index.js",
  "dependencies": {
    "jovo-framework": "^1.0.0"
  },
  "devDependencies": {},
  "scripts": {
    "dev": "jovo run --watch"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/jovotech/jovo-sample-voice-app-nodejs.git"
  },
  "author": "Jovo",
  "license": "Apache-2.0",
  "bugs": {
    "url": "https://github.com/jovotech/jovo-sample-voice-app-nodejs/issues"
  },
  "homepage": "https://github.com/jovotech/jovo-sample-voice-app-nodejs#readme"
}
```

package.json for the "old" project

```json
{
  "name": "tv-helper",
  "version": "1.0.0",
  "main": "index.js",
  "scripts": {
    "dev": "jovo run --watch"
  },
  "dependencies": {
    "array-shuffle": "^1.0.1",
    "aws-sdk": "^2.210.0",
    "axios": "^0.18.0",
    "body-parser": "^1.18.2",
    "cross-env": "^5.1.4",
    "express": "^4.16.2",
    "jovo-framework": "^1.0.4",
    "node-fetch": "^2.0.0"
  },
  "devDependencies": {
    "jovo-cli": "^1.0.9",
    "nodemon": "^1.17.1"
  },
  "author": "Red Ventures"
}
```

**Comment by jankoenig** [comment]
It downloads `^1.0.0`, which means the highest version >= 1.0 and > 2.0.
If you take a look at `node_modules/jovo-framework/package.json`, it should show `"version": "1.1.1"`. So the custom logging should work.

I keep thinking this is not the issue of the framework but rather a different configuration problem? Could you tell me why you have `jovo-cli`and `nodemon`as devDependencies? Maybe this is causing a clash with the global jovo CLI? (sorry, just trying to be creative here, it's difficult to debug without knowing how your code looks like, and since the fresh version works, I feel like there's a possibility it isn't a problem with the framework)

**Comment by ekdevdes** [comment]
No worries man, thanks for all your help! The thinking was that if someone new to our project comes onto our team and doesn’t have the jovo cli or nodemon they can just run “npm install” to get all the needed packages. 

Do you think I should remove them as dev deps and add a postinstall script in the package.json that does a “npm i -g jovo-cli nodemon”?

**Comment by jankoenig** [comment]
This might be a good idea to test

**Comment by rv-ekramer** [comment]
Ok. I’ll try that out. Thanks.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by rv-ekramer** [comment]
Sorry @jankoenig no luck.

**Comment by rv-ekramer** [comment]
Sorry for the late response, I've been busy with a few personal matters. I just ran `rm -rf node_modules` in the directory of the skill I have already created and then ran `npm uninstall -g jovo-cli` and `npm uninstall -g nodemon`. To make sure they were uninstalled I ran `which jovo` and `which nodemon`. When I ran `which jovo` after uninstalling the cli globally I did not get any response. After running `which nodemon` after uninstalling nodemon globally I got `usr/local/bin/nodemon` (which I found to be a little weird since I just uninstalled it). Anywho,I ran `npm i` again after inside of my project folder and with the dependencies in my `package.json` and the `postinstall` script I have to install nodemon and the jovo-cli globally, after which time I had jovo cli and nodemon installed at `/Users/ekramer/.nvm/versions/node/v8.9.4/lib/node_modules/`, I tried running `npm run dev` (which contains the following: `cross-env NODE_ENV=development jovo run --watch` and just `jovo run --watch` by itself and neither worked :(.
